Makefile

```diff
@@ -129,6 +129,26 @@ test-coverage:
 	@export VERSION=$(VERSION); bash -x scripts/test_cover.sh
 .PHONY: test-coverage
 
+## txsim-install: Install the tx simulator/fuzzer.
+txsim-install:
```

Comment:
[blocking] I don't think this command should be contain the `-install` suffix because it doesn't install the binary. It builds the binary. From the help docs:

```
$ go help build
usage: go build [-o output] [build flags] [packages]

Build compiles the packages named by the import paths,
along with their dependencies, but it does not install the results.
```

Reply:
🎯 thanks for the rigorous review! Edit to go install instead of build